File: C sharp/1-50/10.cs
using System;
class Program
{
    static void Main()
    {
        long A, B, C, N, d;
        N = long.Parse(Console.ReadLine());
        string[] input = Console.ReadLine().Split();
        A = long.Parse(input[0]);
        B = long.Parse(input[1]);
        C = long.Parse(input[2]);
        d = A + B + C;
        if (N <= d)
        {
            Console.WriteLine("Yes");
        }
        else
        {
            Console.WriteLine("No");
        }
    }
}

File: Dart/1-50/10.dart
import 'dart:io';
void main() {
  late int A, B, C, N, d;
  N = int.parse(stdin.readLineSync()!);
  List<String> input = stdin.readLineSync()!.split(' ');
  A = int.parse(input[0]);
  B = int.parse(input[1]);
  C = int.parse(input[2]);
  d = A + B + C;
  if (N <= d) {
    print('Yes');
  } else {
    print('No');
  }
}
